CHANGELOG.md

```diff
@@ -1,17 +1,11 @@
-## <sub>v4.0.3-alpha.1</sub>
+## <sub>v4.0.3</sub>
 
-#### _Feb. 24, 2021_
+#### _Feb. 26, 2021_
 
 **Bugfixes**
 
 - 🧭 Fixed border colour of active nav buttons
 - 🎯 +/- icons are vertically centred
```

Comment:
would you mind fixing my mistake and adding `Targeted content:` in here. sorry thanks